src/utils/formUtils.tsx

```diff
@@ -18,6 +21,28 @@ export const findValueFromFormData = (
   return formData.find(({ field }) => field === fieldName)?.value ?? '';
 };
 
+export const validateFormInputField = (
+  formData: FormDataItem[],
+  fieldName: string
+): string | null => {
+  const value = formData.find(({ field }) => field === fieldName)?.value ?? '';
+  if (typeof value === 'string') {
```

Comment:
Won't this always be a string if you're doing nullish coalescing? I would just remove the operator and just do:
`if (!value) {
    return null;
}
...... switch statement`

Reply:
So return early and then do the switch statement after if value exists

Reply:
The `value` variable has the potential to also be a boolean or number, since it is using the `value` property within the `FormDataItem` type:

![image](https://github.com/CapgeminiInventUK/donate-to-educate-frontend/assets/10779091/70ea5307-3813-4e3a-ac0d-7b0db3ad78b1)

which is why I added the typeguard (since the ticket only mentioned to add validation to email addresses and phone numbers, both of which are handled as strings). I think within the charity sign up the values will always be strings as you said, but perhaps there are other forms where this is not the case. I'm still happy to try modify this code if you think it could be cleaner though.